packages/gatsby-mdx/utils/get-source-plugins-as-remark-plugins.js

```diff
@@ -91,11 +91,13 @@ module.exports = async function getSourcePluginsAsRemarkPlugins({
           );
           return markdownAST;
         };
+
+      return [wrappedPlugin, {}];
```

Comment:
Making sure mdPlugins are handed to mdx/remark/unified with explicit options (ie. `[plugin, {}]`) rather than just plugin. Seems unified will only wait for promises if options are passed in as well.

Reply:
wow that's bizarre, good catch.